src/librbd/ImageCtx.cc

```diff
@@ -801,13 +802,17 @@ class SafeTimerSingleton : public SafeTimer {
     ASSIGN_OPTION(mtime_update_interval, uint64_t);
     ASSIGN_OPTION(atime_update_interval, uint64_t);
     ASSIGN_OPTION(skip_partial_discard, bool);
+    ASSIGN_OPTION(discard_granularity_bytes, uint64_t);
```

Comment:
Only two integer choices w/ the config option --- ```uint64_t``` and ```int64_t``` since it's stored as a ```boost::variant``` [1]

[1] https://github.com/ceph/ceph/blob/master/src/common/options.h#L133